.github/test-spec.yml

```diff
@@ -1,6 +1,6 @@
 # This is the Jenkins ci variant of the .github/labler.yaml
 
-"CI-run-twister":
```

Comment:
Please do not change this name. Some CI feature relies on this naming. In case of CI issues please open ticket in helpdesk

Reply:
https://ci-help.nordicsemi.no/#ticket/zoom/3679